### ISSUE TITLE
Screen Reader: Cannot adjust Footer Credit options

### ISSUE THREAD

**Issue by mreishus** [Automattic/wp-calypso#43686]
#### Steps to reproduce

On an iPhone using Safari [with VoiceOver active](https://support.apple.com/guide/iphone/turn-on-and-practice-voiceover-iph3e2e415f/ios),

Starting at: My Site → Design → Customize → Site Identity, scroll to the bottom.

Make sure iOS VoiceOver is active.
There will be a "Footer Credit:" label with a dropdown.  
Double click the dropdown  to adjust the dropdown.
An IOS selection widget, allowing you to adjust the value of the dropdown should appear in the bottom part of the screen.
Make quick swipes up and down with one finger to adjust the value.

#### What I expected

I expected to be able to freely adjust the dropdown to any value in it.

#### What happened instead

After a few swipes, Safari will crash.  It took me a while to make it crash the first time; a few swipes worked, but it eventually crashed.  After the first crash, when re-opening the dropdown, the initial swipe always makes safari crash.

#### Browser / OS version

iPhone X.  IOS 13.5.1.  Safari browser used with VoiceOver accessibility turned on.

#### Screenshot / Video

Video:
[voiceover-footer-credit.MP4.zip](https://github.com/Automattic/wp-calypso/files/4831992/voiceover-footer-credit.MP4.zip)

#### Context / Source

#3030290-zen

**Comment by youbrokesomething** [comment]
This isn't something that I can reproduce using an emulator, so I've asked my team to see if we can reproduce it on a real iPhone.
There have been many updates to customizer since this ticket was opened; this issue may no longer be reproducible.

**Comment by youbrokesomething** [comment]
We can't reproduce this issue on emulators or a real iPhone device. Software changes over time likely resolved this issue. I'm going to close this issue. We should open a new one if we get new reports.
